server/middleware/authentication/verifyUser.js

```diff
@@ -0,0 +1,20 @@
+const { verify } = require('../../utils/jwt');
+const boomify = require('../../utils/boomify');
+
+const verifyUser = async (req, res, next) => {
+  const { token } = req.cookies;
+  if (token) {
+    try {
+      const { decoded } = await verify(token);
+      req.userId = decoded;
```

Comment:
Shouldn't this be `userId` ?
```suggestion
      const { userId } = await verify(token);
      req.userId = userId;
```